urllib3/util/retry.py

```diff
@@ -19,6 +19,10 @@
 
 log = logging.getLogger(__name__)
 
+
+DEFAULT_REDIRECT_HEADERS_BLACKLIST = ['Authorization']
```

Comment:
Would it be better to put this under the `Retry` object next to `RETRY_AFTER_STATUS_CODES` etc as a frozenset too?

(Sorry I should have mentioned that before)

Reply:
I was looking for globals and didn't find them! Didn't think to look at the bottom of the module. :)